.github/workflows/generate-artifacts.yml

```diff
@@ -0,0 +1,69 @@
+name: allcontributors-zenodo
+
+on:
+  pull_request: []
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Setup test environment
+        run: conda create --quiet --name test requests
+
+      - name: Run Testing
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          export PATH="/usr/share/miniconda/bin:$PATH"
+          source activate test
+          /bin/bash tests/test_client.sh
+
+      - name: Generate Updated Zenodo and Contributors
+      
+        # Important! Update to release https://github.com/con/allcontributors-zenodo
+        uses: ./
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # ".zenodo.json to update. If does not exist, must define zenodo_doi"
+          zenodo_file: .zenodo.json
+
+          # Zenodo DOI needed for init. Leave unset to skip init.
+          zenodo_doi: 10.5281/zenodo.1012531
+
+          # INFO, DEBUG, ERROR, WARNING, etc.
+          log_level: DEBUG
+
+          # the minimum number of contributions required to add a user
+          threshold: 1
+
+          # Contribution type, which defaults to "code" if not set.
+          ctype: code
+
+          # skip running all-contributors generate
+          skip_generate: false
+
+      - name: Upload zenodo data as artifact
+        uses: actions/upload-artifact@v2-preview
+
+        # Path is relative to GITHUB_WORKSPACE
+        with:
+          name: zenodo
+          path: .zenodo.json
+
+      - name: Upload allcontributors data as artifact
+        uses: actions/upload-artifact@v2-preview
+        with:
+          name: allcontrib
+          path: .all-contributorsrc
```

Comment:
as mentioned in the https://github.com/con/allcontributors-zenodo/issues/1#issuecomment-650858118 comment, I do not think we should touch this or README.md -- those are under allcontributors control.  The main goal is to get .zenodo.json (or whatever next useful thing comes about ;-))

Reply:
The reason it's uploaded is because it's an example README with those sections that are generated by all contributors. Our library doesn't do anything to it, but the action uses the command line tool for all contributors to generate the readme. Thus, for the action we would want to check the generated readme.

Reply:
rright -- this is done by allcontributors, good!